src/v2/Apps/Fair/__tests__/FairApp.jest.tsx

```diff
@@ -126,32 +120,21 @@ describe("FairApp", () => {
       }),
     })
 
-    fireEvent.click(screen.getByText("Booths"))
+    fireEvent.click(screen.getByText("Exhibitors A-Z"))
```

Comment:
**question**: Should we remove this click if we're testing click tracking to the artworks tab in this assertion? Having the context_module remain `fairInfo` seems fine to me - unless I'm missing something.

Reply:
Done